packages/dds/tree/src/test/shared-tree/sharedTree.spec.ts

```diff
@@ -2083,14 +2083,14 @@ const testValueSchema = namedTreeSchema({
  * Inserts a single node under the root of the tree with the given value.
  * Use {@link peekTestValue} to read the value.
  */
-function pushTestValue(checkout: ISharedTreeCheckout, value: TreeValue): void {
+function pushTestValue(checkout: ISharedTreeBranch, value: TreeValue): void {
 	insert(checkout, 0, value);
 }
```

Comment:
```suggestion
function pushTestValue(branch: ISharedTreeBranch, value: TreeValue): void {
	insert(branch, 0, value);
}
```

Reply:
Or, tree to be consistent with the functions below:

```suggestion
function pushTestValue(tree: ISharedTreeBranch, value: TreeValue): void {
	insert(tree, 0, value);
}
```